src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/aws/AWSEC2JCloudsProvider.java

```diff
@@ -58,6 +62,8 @@
 @Component
 public class AWSEC2JCloudsProvider extends JCloudsProvider {
 
+    private static final Logger logger = Logger.getLogger(AWSEC2JCloudsProvider.class);
+
     @Getter
     private final String type = "aws-ec2";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1170)